### PR TITLE
Add export_flag to ProjectProperties; populate from XER import; allow "Y" and "Yes" as valid string booleans

### PR DESCRIPTION
**Pull request by forenpm**
Modified ProjectProperties to include EXPORT_FLAG, which denotes when a project was chosen for export in P6.

**Reason**: When auto-integrating projects from a multi-project file with external/integrated relationships, I need to ignore projects that were automatically included by P6, but not specifically set for export by the user.

Modified getBoolean implementation to accept "y"/"Y" and "yes"/"YES" as valid booleans.  This may be an overstep, but many of the flags are simply "Y" in XER.

### CODE REVIEW

File: src/net/sf/mpxj/primavera/PrimaveraReader.java [hunk 1]
```diff
@@ -1746,7 +1747,10 @@ private Number calculateDurationPercentComplete(Row row)
       RESOURCE_TYPE_MAP.put(null, ResourceType.WORK);
       RESOURCE_TYPE_MAP.put("RT_Labor", ResourceType.WORK);
       RESOURCE_TYPE_MAP.put("RT_Mat", ResourceType.MATERIAL);
-      RESOURCE_TYPE_MAP.put("RT_Equip", ResourceType.WORK);
+      // Changed RT_Equip from the original (ResourceType.WORK)
+      // I need to be able to detect RT_Equip
+      // This will probably present problems if I use other methods like time phasing, or anything else that hits this enum in MPXJ
```
Comment (joniles) [inline src/net/sf/mpxj/primavera/PrimaveraReader.java]: Can you elaborate on potential issues? Could we move this to a separate PR as I was about to merge the other change. Thanks!

Comment (forenpm) [reply src/net/sf/mpxj/primavera/PrimaveraReader.java]: Oops, I forgot that PRs get updated with later check-ins.  I should have done the PR from a branch...

That was really just a note to myself to remember to check into ramifications should I start using time phasing or anything else that might depend on the resource type flag.  I don't know the code well enough to know what impacts that change may have.
